src/leaf-node-xyzaxis.cpp

```diff
@@ -95,9 +95,8 @@ namespace graphics {
         geode_ptr_->addDrawable(sphere_drawable_ptr_);
         if(sizeAxis_ > 0){  // optimisation of memory consumption : doesn't create the axis instead of creating axis with size "0"
              /* create the axis : */
-            // float radiusCyl = (getRadius()/4.f) * getSizeAxis();
             float radiusCyl = getRadius()/4.f;
-            float lengthCyl = getSizeAxis();
+            float lengthCyl = (getRadius()*3.f) * getSizeAxis();
```

Comment:
Hum, I like the idea that the size axis is the length of the cylinders. Why don't you pass a different axis size ?

Reply:
Ok the goal of this change was to keep the same aspect ratio between the sphere and the arrows of the markers when you change the radius of the sphere. But I agree that the names will not be relevant anymore after this change. 

May be we could change and simplify the python API and replace "radiusSphere" and "sizeAxis" with a single "Scale" argument. I do not see any use-case where you want to specify one of them separately ?

Anyway, you can close this PR.